homeassistant/components/rainbird/sensor.py

```diff
@@ -1,44 +1,39 @@
 """Support for Rain Bird Irrigation system LNK WiFi Module."""
 import logging
 
-import voluptuous as vol
+from pyrainbird import RainbirdController
 
-from homeassistant.components.sensor import PLATFORM_SCHEMA
-from homeassistant.const import CONF_MONITORED_CONDITIONS
-import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.entity import Entity
 
-from . import DATA_RAINBIRD
-
-_LOGGER = logging.getLogger(__name__)
+from . import (
+    DATA_RAINBIRD,
+    RAINBIRD_CONTROLLER,
+    SENSOR_TYPE_RAINDELAY,
+    SENSOR_TYPE_RAINSENSOR,
+    SENSOR_TYPES,
+)
 
-# sensor_type [ description, unit, icon ]
-SENSOR_TYPES = {"rainsensor": ["Rainsensor", None, "mdi:water"]}
+PARENT_SENSOR = "parent_sensor"
```

Comment:
Where is this used?

Reply:
Not used now. Was for pairing binary_sensor to sensor. Removing.